components/Utilities/SharedBus/SharedBus.c

```diff
@@ -3,15 +3,31 @@
 #include "Custom_Log.h"
 
 #define BIT_23	( 1 << 23 ) //prevent overwriting bus for send 
-#define BIT_22	( 1 << 22 ) //permisson for all components to peek from SharedBus 
-#define BIT_21	( 1 << 21 ) //check if received the packet ones
+#define BIT_22	( 1 << 22 ) //permission for all components to peek from SharedBus 
+
+#define UI_DAEMON_ID        ( 1 << UI_INTERFACE_ID )
+#define MATTER_DAEMON_ID    ( 1 << MATTER_INTERFACE_ID ) 
+#define MQTT_DAEMON_ID      ( 1 << MQTT_INTERFACE_ID )
```

Comment:
this section is breaking the modularoty of the code. SharedBus should not need an update when you want to add new service to your application